public/js/ajax-airline-handler.js

```diff
@@ -0,0 +1,151 @@
+$('form').submit(function(event){
+    event.preventDefault();
+})
+
+const createAirline = () => {
+    fetch('api/airlines', {
+        method: 'POST',
+        headers: {
+            'Content-type': 'application/x-www-form-urlencoded',
+            'X-Requested-With': 'XMLHttpRequest',
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        },
+        body: new URLSearchParams($('#add-airline-form').serialize())
+    })
+
+    .then(response => {
+        if (response.ok) {
+            $('#name-error').text('');
+            $('#description-error').text('');
+            document.forms["add-airline-form"].reset();
+
+            return response.json();
+
+        } else {
+            return response.json().then(data => {
+                handleValidationErrors(data.errors);
+            });
+        }
+    })
+    .then(result => {
+        loadTable();
+    })
+    .catch(error =>
+        console.error(response.message));
+}
+
+const deleteAirline = (airlineId) => {
+    fetch(`api/airlines/${airlineId}`, {
+        method: 'DELETE',
+        headers: {
+            'X-Requested-With': 'XMLHttpRequest',
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        },
+    })
+
+    .then(response => response.json())
+    .then(result=>
+        loadTable())
+    .catch(error =>
```

Comment:
```suggestion
    .catch(error => console.error)
```
I'm not entirely sure about this, but I think that doing just `console.error` equals to `console.error(error)`. Lemme know if it works.